data/avro-schema/src/main/resources/avro/net/corda/data/membership/p2p/MembershipRegistrationRequest.avsc

```diff
@@ -21,6 +21,11 @@
       "name": "memberSignature",
       "doc": "Signature of the member over the memberContext.",
       "type": "net.corda.data.crypto.wire.CryptoSignatureWithKey"
+    },
+    {
+      "name": "isPending",
```

Comment:
Why do we need this on a registration request? Aren't all registration requests pending when sent from member to MGM?

Reply:
Unfortunately, we are persisting the signatures from the request and in a different table, but that table is using the same keys as the member info one and queries for them during distribution. As a temporary solution until we do https://r3-cev.atlassian.net/browse/CORE-10689 i put the signatures as pending and then update the is_pending field once the members are approved, so that their signatures can be used.

Reply:
Ok, sounds fine as long as it will be updated in CORE-10689

Reply:
Hmm, ok. Doesn't seem ideal but if it will be fixed in CORE-10689 it's not too bad.